app/helpers/publish_courses_helper.rb

```diff
@@ -9,14 +9,7 @@ def apply_course_and_route_summary(course)
     t(".summary_with_route", summary: apply_course_summary(course), route: t("activerecord.attributes.trainee.training_routes.#{course.route}"))
   end
 
-  def course_summary_text_for(trainee, course)
-    summary_with_route = t(".summary_with_route", summary: course.summary, route: t("activerecord.attributes.trainee.training_routes.#{course.route}"))
-    trainee.apply_application? ? summary_with_route : course.summary
-  end
-
-  def courses_fieldset_text_for(trainee)
-    return t("views.forms.publish_course_details.all_courses_message") if trainee.apply_application?
-
-    t("views.forms.publish_course_details.route_message", route: route_title(trainee.training_route))
+  def courses_fieldset_text_for(_trainee)
```

Comment:
You can get rid of the `_trainee` argument. You'll have to update the method here also: http://github.com/DFE-Digital/register-trainee-teachers/blob/f2f6f3a83e5be4b366629cd2ca09a868d2bec962/app/views/trainees/publish_course_details/edit.html.erb#L29-L29